ui/tests/acceptance/client-detail-test.js

```diff
@@ -17,6 +17,10 @@ let clientToken;
 
 const wasPreemptedFilter = allocation => !!allocation.preemptedByAllocation;
 
+function nonSearchPOSTS() {
+  return server.pretender.handledRequests.reject(request => request.url.includes('fuzzy')).filterBy('method', 'POST');
+}
+
```

Comment:
Seeing stuff like this always makes me want to write some sort of lib that makes building this type of behavior on top of Pretender easier.